peps/pep-0757.rst

```diff
@@ -238,6 +238,17 @@ Implementation
 Benchmarks
 ==========
 
+Code::
+
+    /* Query parameters of Python’s internal representation of integers. */
+    const PyLongLayout *layout = PyLong_GetNativeLayout();
+
+    size_t int_digit_size = layout->digit_size;
+    size_t int_digits_order = layout->digits_order;
+    size_t int_bits_per_digit = layout->bits_per_digit;
```

Comment:
Oops, perhaps I should use GMP-like terminology for variables (int_limb_size, etc).